build.gradle

```diff
@@ -16,13 +16,13 @@ plugins {
     id 'java'
     id 'java-library'
     id 'application'
-    id "org.owasp.dependencycheck" version "8.1.2"
+    id "org.owasp.dependencycheck" version "8.2.1"
```

Comment:
What about a plugin to collect git information? Shouldn't we add it as well?

Reply:
Testing changes in CI is in progress

Reply:
@OptimumCode could you please take a look https://github.com/th2-net/.github/pull/32/files